tools/test-setup.sh

```diff
@@ -4,8 +4,9 @@ set -euxo pipefail
 
 sudo dd if=/dev/zero of=/swap.img bs=1024 count=1048576
 sudo chmod 600 /swap.img
-sudo mkswap /swap.img
-sudo swapon /swap.img
+sudo losetup -f /swap.img
+sudo mkswap $(sudo losetup --associated swap.img|sed 's,:.*,,')
```

Comment:
hm. I guess you're missing a '/' here, leading to the error:
```
mkswap: error: Nowhere to set up swap on?
```

(btw, thanks for looking at this, I failed to find time to work on this swap thing so far)

Reply:
Is this going to work even if we run on systems w/o btrfs? I really want to keep test-setup.sh as "portable" as possible.

Reply:
I've not tested it but I guess that it should work on local filesystems.

Reply:
Do you really need this step?

Reply:
the tests are creating one or several VMs and it's asking some memory. iirc, the test suite got issues in the past due to this, so adding a swap file helps.